frontend/src/components/ToolTip/ToolTip.tsx

```diff
@@ -0,0 +1,28 @@
+import { FunctionComponent } from 'react';
+import { ToolTipText, ToolTip } from './ToolTip.style';
+import { colorPalette } from 'stylesheet';
+
+interface Props {
+  children?: any;
+  toolTipText?: string;
+  bottom ?: string;
+  color ?: string;
+  backgroundColor ?: string;
+  reverse ?: boolean;
```

Comment:
Perhaps `reverse` is a rather ambiguous name. For a tooltip component, one might think that it would change the position of the tooltip for example. I think a name like `invertColor` is more explicit.

Reply:
You are now using the prop `invertPosition`, I think that is even more ambiguous, isn't it?

Reply:
It's because now we have 2 types of tooltips : 
![image](https://user-images.githubusercontent.com/99249234/173014542-e7b80fed-4468-4fee-bf26-4c8126024dcf.png)
![image](https://user-images.githubusercontent.com/99249234/173014557-a8d95454-225f-42e3-836c-c5d4e3a53d00.png)

I rename this variable for onBottom (true or false)